code/game/mecha/working/ripley.dm

```diff
@@ -180,4 +180,13 @@
 	else
 		step_in = 5
 		for(var/obj/item/mecha_parts/mecha_equipment/drill/drill in equipment)
-			drill.equip_cooldown = initial(drill.equip_cooldown)
\ No newline at end of file
+			drill.equip_cooldown = initial(drill.equip_cooldown)
+
+/obj/mecha/working/ripley/emag_act(mob/user)
+	if(!emagged)
+		emagged = TRUE
+		to_chat(user, "<span class='notice'>You slide the card through [src]'s ID slot.</span>")
+		playsound(loc, "sparks", 100, 1)

```

Comment:
vol 100 seems pretty high! Everything else tends to cap out at 75... with most at 60.....

Reply:
See above, not something I changed, I just moved it.